cmd/mothership/mothership/start/scheduler.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func startScheduler(ctx context.Context, o *Options) error {
 
-	runtimeBuilder := service.NewRuntimeBuilder(o.Registry.ReconciliationRepository(), logger.NewLogger(o.Verbose))
+	runtimeBuilder := service.NewRuntimeBuilder(o.Registry.ReconciliationRepository(), logger.NewLogger(o.Verbose), o.Verbose)
```

Comment:
I think here you are submitting the Verbose Flag twice. I suggest to omit this and instead use the Logger as intended and switch all log messages to debug unless they really need to be on info level. Then you won't have this issue anymore

Reply:
Regarding the original issue of client-go, I suggest we mask the Log with our own, overwriting the Level in this case.